site/layouts/mobile/list.html

```diff
@@ -0,0 +1,27 @@
+{{ define "main" }}
+
+    <div class="pv6 ph2 tc-l">
+        <a class="f6 dim br-pill ph3 pv2 ma2 dib white bg-primary">Beta</a>
+            <h1 class= "f1 b">Hire on the go.</h1>
+                <h2 class="f4">No matter where you are, you can hire with enlist's iOS and Android apps.</h2>
+                    <ul class="pa4 tl ml7">
+                        <li>&check;Find what needs your attention quickly</li>
+                        <li>&check;Search for the candidates you want</li>
+                        <li>&check;Access all your information anytime, anywhere</li>
+                        <li>&check;Collaborate with your team on the go</li>
+                    </ul>
+        
+        <div class="flex flex-wrap mhn1">
+            <div class="fl w-50-ns pl7">
+                <a href="https://apps.apple.com/app/id1146472597">
+                    <img alt="Download from the app store" src="/img/app-store-download.svg" style="width:220px;height:100px;">
```

Comment:
I suggest removing the `style` parameter from the `img` element and instead adding `class="mw-100"` to the `img' element.